.github/workflows/checks.yml

```
@@ -22,9 +22,33 @@ jobs:
+        run: |
+          touch .env
+          echo DB_HOST="${{ secrets.DB_HOST }}" >> .env
+          echo DB_NAME="${{ secrets.DB_NAME }}" >> .env
+          echo DB_USER="${{ secrets.DB_USER }}" >> .env
+          echo DB_PASSWORD="${{ secrets.DB_PASSWORD }}" >> .env
+          echo DJANGO_SECRET_KEY="${{ secrets.DJANGO_SECRET_KEY }}" >> .env
+          cat .env
+          pwd
+
       - name: Checkout
         uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)